### PR TITLE
Added support for work in browser

### PR DESCRIPTION
**Pull request by daimor**
For cases like github.dev and vscode.dev, following the [recommendations](https://code.visualstudio.com/api/extension-guides/web-extensions) 
Very basic support, only coloring, and symbol providers

image

### CODE REVIEW

**Comment by daimor** [comment]
@gjsjohnmurray what do you think about this PR?

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
   "engines": {
     "vscode": "^1.43.0"
   },
-  "enableProposedApi": true,
+  "enableProposedApi": false,
```
Comment (gjsjohnmurray) [inline package.json:52]: Why this change? I thought we set it true so beta builds work with proposed API, then the release action changes it to false. I know this property is going away at 1.63, but at the moment we declare 1.43 as our minimum version. And 1.63 hasn't shipped yet.

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (daimor) [reply package.json]: yeah, returned it back

**Review by gjsjohnmurray — APPROVED** [review]
